lake/utils/parse_clkwork_config.py

```diff
@@ -145,7 +132,10 @@ def map_controller(controller, name):
         print()
 
     # Now transforms ranges and strides
-    (tform_extent, tform_cyc_strides) = transform_strides_and_ranges(ctrl_ranges, ctrl_cyc_strides, ctrl_dim)
+    tform_cyc_strides = None
+    if ctrl_cyc_strt is not None:
```

Comment:
What happens if it is `None`?

Reply:
If it's None, we don't call transform_strides_and_ranges on a controller that doesn't exist, which will throw some error. However, I don't remember what caused the need for this when we were testing with Joey before...I think the controller didn't exist for some reason, but that doesn't seem to make sense now...